Src/java/cql-to-elm-cli/src/main/java/org/cqframework/cql/cql2elm/cli/Main.java

```diff
@@ -91,12 +92,21 @@ private static void writeELM(
             NpmPackageManager pm = new NpmPackageManager(igContext.getSourceIg());
             pm.getNpmList().forEach(npm -> {
                 NamespaceInfo newNamespace = new NamespaceInfo(npm.id(), npm.canonical());
-                namespaceManager.ensureNamespaceRegistered(newNamespace);
+                if (namespaceManager.resolveNamespaceUri(newNamespace.getName()) != null) {
+                    // TODO: Logger.skip loading
+                } else if (namespaceManager.getNamespaceInfoFromUri(newNamespace.getUri()) != null) {
+                    // TODO: Logger.skip loading
+                } else {
+                    namespaceManager.ensureNamespaceRegistered(newNamespace);
+                }
             });
             LibraryLoader reader = new LibraryLoader(igContext.getFhirVersion());
             NpmLibrarySourceProvider sp = new NpmLibrarySourceProvider(pm.getNpmList(), reader, pm);
             libraryManager.getLibrarySourceLoader().registerProvider(sp);
 
+            NpmModelInfoProvider mp = new NpmModelInfoProvider(pm.getNpmList(), reader, pm);
```

Comment:
Question:  is this to address the issue in question or to add missing functionality?